pkg/tlsutil/tlsutil.go

```diff
@@ -102,7 +102,7 @@ func ParsePEMEncodedPrivateKey(pemdata []byte) (*rsa.PrivateKey, error) {
 	return x509.ParsePKCS1PrivateKey(decoded.Bytes)
 }
 
-func NewSignedCertificate(cfg CertConfig, key *rsa.PrivateKey, caCert *x509.Certificate, caKey *rsa.PrivateKey) (*x509.Certificate, error) {
+func newSignedCertificate(cfg CertConfig, key *rsa.PrivateKey, caCert *x509.Certificate, caKey *rsa.PrivateKey, usage []x509.ExtKeyUsage) (*x509.Certificate, error) {
```

Comment:
nit: `newSignedClientAndServerCertificate` to keep consistent with `newSignedClientCertificate`